evap/staff/templates/staff_course_type_index.html

```diff
@@ -12,7 +12,7 @@
         <a href="{% url 'staff:course_type_merge_selection' %}" class="btn btn-sm btn-secondary">{% trans 'Merge course types' %}</a>
     </div>
 
-    <form id="course-type-form" method="POST" enctype="multipart/form-data" class="form-horizontal select2form">
```

Comment:
(not related to this line)

When entering something in the "import names" form on course type index or degree index and hitting "tab" (as we will be teaching our staff users), I accept the suggested tag but also tab to the next input

Reply:
Maybe we should just intercept enter submitting the form, but not teach anything specifically? It would do the right thing: Enter only submits if you typed an option that already is a tag.